src/judgeReceiver.js

```diff
@@ -103,23 +109,24 @@ var startVote = (timeoutInSeconds) => {
     votesNotGuilty = 0;
     seekingVerdict = true;
     onVotesChanged();
-    timeElapsed = setTimeout(() => {
-        endVote();
-    }, (timeoutInSeconds * 1000));
-};
-
-var voteTimeout = async (timeoutInSeconds) => {
-
+    timeElapsed = 0;
+    timer = setInterval(() => {
+        timeElapsed++;
```

Comment:
Maybe instead of counting up, you should count down?

Reply:
That way if we want to represent time remaining we just need to know starting time, and remaining time?

Reply:
Good call, I'll make that change